### DISCUSSION TITLE
Support for Search pseudo x customization

### DISCUSSION THREAD

**Discussion by chiefpansancolt** [tailwindlabs/tailwindcss#2883] [Ideas]
I think it would be good to add the ability to easily support the x when dark mode is used for input search type, the x is not visible when colors are dark.

**Comment by MathGeniusJodie** [comment]
that's more of a job for the tailwind forms plugin
https://github.com/tailwindlabs/tailwindcss-forms

**Reply by chiefpansancolt** [reply]
good call! I created an issue on that repo
